src/test/java/io/jenkins/plugins/analysis/core/scm/GitBlameITest.java

```diff
@@ -0,0 +1,200 @@
+import hudson.model.FreeStyleProject;
+import hudson.model.Item;
+import hudson.model.Result;
+import hudson.plugins.git.GitSCM;
+import jenkins.plugins.git.GitSampleRepoRule;
+
+import io.jenkins.plugins.analysis.core.model.AnalysisResult;
+import io.jenkins.plugins.analysis.core.testutil.IntegrationTestWithJenkinsPerSuite;
+import io.jenkins.plugins.analysis.warnings.Java;
+import io.jenkins.plugins.analysis.warnings.recorder.pageobj.SourceControlRow;
+import io.jenkins.plugins.analysis.warnings.recorder.pageobj.SourceControlTable;
+
+import static io.jenkins.plugins.analysis.core.assertions.Assertions.*;
+import static java.nio.file.StandardOpenOption.*;
+
+/**
+ * Integration test for GitBlamer.
+ *
+ * @author Andreas Neumeier
+ * @author Tobias Redl
+ */
+public class GitBlameITest extends IntegrationTestWithJenkinsPerSuite {
+    private static final String JAVA_FILE = "Hello.java";
+    private static final String CPP_FILE = "Hello.cpp";
+    private static final String USER_1_EMAIL = "John@localhost";
+    private static final String USER_1_AUTHOR = "John Doe";
+    private static final String USER_2_EMAIL = "Jane@localhost";
+    private static final String USER_2_AUTHOR = "Jane Doe";
+    private static final String JAVA_WARNING = "Unexpected end of File";
+
+    /**
+     * Git repository used for Testing.
+     */
+    @Rule
+    public GitSampleRepoRule gitRepo = new GitSampleRepoRule();
+
+    /**
+     * Test if Source Control Table contains a git blame issue when a simple Java syntax error occurs.
+     */
+    @Test
+    public void shouldShowOneGitBlameWarning() {
+        FreeStyleProject project = createScmJavaFreestyleProject();
+        appendTextToFileInScm(gitRepo, JAVA_FILE, "public class HelloWorld {\n"
+                + "       public static void main (String[] args) {\n"
+                + "             System.out.println(\"Hello World!\");\n"
+                + "       }}", USER_1_AUTHOR, USER_1_EMAIL);
+        appendTextToFileInScm(gitRepo, JAVA_FILE, "{", USER_2_AUTHOR, USER_2_EMAIL);
+        saveJavaWarningToScm(gitRepo, JAVA_FILE, 4, JAVA_WARNING);
+
+        AnalysisResult analysisResult = scheduleBuildAndAssertStatus(project, Result.SUCCESS);
+        assertThat(analysisResult).hasTotalSize(1);
+
+        HtmlPage detailsPage = getDetailsWebPage(project, analysisResult);
+        SourceControlTable sourceControlTable = new SourceControlTable(detailsPage);
+
+        List<SourceControlRow> sourceControlRows = sourceControlTable.getRows();
+        assertThat(sourceControlRows.get(0).getValue(SourceControlRow.AUTHOR)).isEqualTo(USER_2_AUTHOR);
+        assertThat(sourceControlRows.get(0).getValue(SourceControlRow.EMAIL)).isEqualTo(USER_2_EMAIL);
+        assertThat(sourceControlRows.get(0).getValue(SourceControlRow.FILE)).isEqualTo(JAVA_FILE + ":4");
+        assertThat(sourceControlRows.get(0).getValue(SourceControlRow.DETAILS_CONTENT)).isEqualTo(JAVA_WARNING);
```

Comment:
Hier wäre noch schön gewesen, mehr als ein Ergebnis in der Tabelle zu haben. Auch ein Assert auf die Anzahl der Rows fehlt.